arctic/store/_version_store_utils.py

```diff
@@ -28,9 +30,13 @@ def checksum(symbol, doc):
     Checksum the passed in dictionary
     """
     sha = hashlib.sha1()
-    sha.update(symbol)
-    for k in sorted(doc.iterkeys(), reverse=True):
-        sha.update(str(doc[k]))
+    sha.update(symbol.encode('ascii'))
+    for k in sorted(iter(doc.keys()), reverse=True):
```

Comment:
Can we add a unit test that, given a known document `checksum` returns a know good checksum?